src/rust/engine/logging/src/logger.rs

```diff
@@ -118,12 +118,11 @@ impl Logger {
       })
   }
 
-  pub fn log_from_python(
-    &self,
-    message: &str,
-    python_level: u64,
-    target: &str,
-  ) -> Result<(), String> {
+  /// log_from_python is only used in the Python FFI, which in turn is only called within the
+  /// Python `NativeHandler` class. Every logging call from Python should get proxied through this
+  /// function, which translates the log message into the Rust log paradigm provided by
+  /// the `log` crate.
```

Comment:
Will this still be true with Stu's CFFI change?

Reply:
Yes I made sure when I reviewed his change that the behavior of `log_from_python` didn't change.